attributes/default.rb

```diff
@@ -23,6 +23,12 @@
 #
 default['chef-ingredient']['mixlib-install']['git_ref'] = nil
 
+#
+# Optionally override the omnitruck endpoint
+#
+default['chef-ingredient']['omnitruck']['endoint'] = nil
```

Comment:
I'm trying to understand why chef-ingredient needs to proxy anything through Omnitruck. We should be able to leverage `Mixlib::Install::Backend::Bintray` directly in this code for all products. I'm actually do something like this in my updates to `chef-server-ctl install`. Basically you leverage the backend to resolve a url/checksum and then download with `remote_file` My WIP updated code looks like:
https://gist.github.com/schisamo/83a2a66c270831587025df563a44d518